src/components/PublicGroupList.vue

```diff
@@ -33,18 +33,82 @@
-                                            <font-awesome-icon v-if="editMode" icon="arrows-alt-v" class="action drag me-3" />
-                                            <font-awesome-icon v-if="editMode" icon="times" class="action remove me-3" @click="removeMonitor(group.index, monitor.index)" />
+                                    <details>
+                                        <summary class="summary">
+                                            <div class="col-9 col-md-8 small-padding">
+                                                <div class="info">
+                                                    <font-awesome-icon v-if="editMode" icon="arrows-alt-v" class="action drag me-3" />
+                                                    <font-awesome-icon v-if="editMode" icon="times" class="action remove me-3" @click="removeMonitor(group.index, monitor.index)" />
+
+                                                    <Uptime :monitor="monitor.element" type="24" :pill="true" />
+                                                    {{ monitor.element.name }}
+                                                </div>
+                                            </div>
+                                            <div :key="$root.userHeartbeatBar" class="col-3 col-md-4">
+                                                <HeartbeatBar size="small" :monitor-id="monitor.element.id" />
+                                            </div>
+                                        </summary>
+                                        <!-- Monitor Details -->
+                                        <!-- Ping Detail -->
+                                        <div class="col">
+                                            <h4>Ping</h4>
+                                            <p>({{ $t("Current") }})</p>
+                                            <span class="num">
+                                                <a href="#" @click.prevent="showPingChartBox = !showPingChartBox">
+                                                    <CountUp :value="heartbeatList" />
+                                                </a>
+                                            </span>
+                                        </div>
+                                        <!-- AvgPing Detail -->
+                                        <div class="col">
+                                            <h4>Avg. Ping</h4>
+                                            <p>({{ $t("Current") }})</p>
+                                            <span class="num">
+                                                <a href="#" @click.prevent="showPingChartBox = !showPingChartBox">
+                                                    <CountUp :value="lastHeartBeat" />
+                                                </a>
+                                            </span>
+                                        </div>
```

Comment:
I don't really see the pattern in these. `showPingChartBox = !showPingChartBox` and `:value="lastHeartBeat"` repeat many times, does it actually render things correctly? If so, I don't understand how.

Reply:
testing variable, apologies for the disorder. 
`showPingChartBox` is for showing the ping chart on the backend, this is not needed for the front end just something I was planning on using to match the backend dashboard functionality.

as for `:value="lastHearBeat"` is from testing and does not render at this time.

the Details.vue component is where the majority of this is defined for the backend so I've been trying to work in reverse from there

Reply:
updated to populate correct data.